run_scans.py

```diff
@@ -9,13 +9,6 @@ def launchTestsArgs(tool, slot, link, chamber,vt1=None,vt2=0,perchannel=False,tr
   from subprocess import CalledProcessError
```

Comment:
Instead of this import, let's 
```
from gempython.utils.wrappers import runCommand
```
and replace all the repeated blocks with
```
runCommand(cmd)
```